### DISCUSSION TITLE
Welcome to timezonefinder Discussions!

### DISCUSSION THREAD

**Discussion by jannikmi** [jannikmi/timezonefinder#109] [General]
## 👋 Welcome!
  Let's use Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sheridansmall** [comment]
Hi, I have been tasked with updating time-zone data for tens of thousands of locations as time-zones can change for a particular location. We intend to update these time-zones every 6 months to capture any changes that occur during this period. I have read through your documentation and timezonefinder seems to do what we want i.e., given a latitude and longitude it returns the time-zone for the location. However, in the documentation it does not mention how current the time-zone data is or how it is calculated. Can someone explain for me how current the time-zone data is and how it is calculated please?

**Reply by jannikmi** [reply]
Sure. I briefly mention it here: https://timezonefinder.readthedocs.io/en/latest/3_about.html
Let me know if there are further quesions

**Reply by sheridansmall** [reply]
Thanks for the reply. The link in the reply says "Current data set in use:  precompiled [timezone-boundary-builder](https://github.com/evansiroky/timezone-boundary-builder)". [timezone-boundary-builder](https://github.com/evansiroky/timezone-boundary-builder) says it uses the [timezone database](https://www.iana.org/time-zones). The latest release of timezone-boundary-builder is [2020d](https://github.com/evansiroky/timezone-boundary-builder/releases/tag/2020d) 05 Nov 2020 which I guess corresponds to release 2020d of https://www.iana.org/time-zones. However the latest release of https://www.iana.org/time-zones is 2021e (Released 2021-10-21). It looks like https://github.com/jannikmi/timezonefinder/blob/master/timezonefinder/file_converter.py is used to build the .bin files in https://github.com/jannikmi/timezonefinder/tree/master/timezonefinder which were last built shortly after the timezone-boundary-builder release  (23 Dec 2020).
As I said earlier I have been tasked with updating locations data to have the current time-zone.
Please correct me if I'm wrong but my understanding is that timezone-boundary-builder and timezonefinder are using https://www.iana.org/time-zones release 2020d not 2021e. Is there a way to use release 2021e or shall I contact the author of timezone-boundary-builder?

**Reply by jannikmi** [reply]
All correct. Yes, if you need to use the version `2021e` please create an issue here https://github.com/evansiroky/timezone-boundary-builder/issues
As soon as there is a new version available, I will update this package.

**Reply by sheridansmall** [reply]
@jannikmi Thanks for the information I have raised an issue on timezone-boundary-builder at https://github.com/evansiroky/timezone-boundary-builder/issues/121 and had a very good reply. Thank you for building such a useful package, it is running very fast and producing accurate results.
